alphabase/psm_reader/__init__.py

```diff
@@ -17,6 +18,6 @@
 from alphabase.psm_reader.sage_reader import SageReaderTSV, SageReaderParquet
 
 try:
-    from alphabase.psm_reader.msfragger_reader import MSFraggerPepXML
+    from alphabase.psm_reader.msfragger_reader import MSFraggerPepXML  # noqa: F401 TODO remove import side effects
```

Comment:
This `side effects` is not clear here

Reply:
I now name the side effect in the comment